packages/admin-page/src/footer.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Automattic\Jetpack\AdminPage;
+
+/**
+ * Create and Render the Jetpack Footer
+ */
+class Footer {
+
+	/**
+	 * Build and render the Jetpack page Footer
+	 *
+	 * @return string
+	 */
+	public function render() {
+
+		$jetpack_admin_url = admin_url( 'admin.php?page=jetpack' );
+		$jetpack_about_url = ( apply_filters( 'jetpack_in_dev_mode', false ) )
```

Comment:
Nit: We don't need the wrapping parens here, do we?

Reply:
Yeah this really looks like something that would look better as a typical `if` statement.

Reply:
I'm fine with the filter, I was just referring to the parens.